math/Matrix Multiplication/matmul.c

```diff
@@ -0,0 +1,81 @@
+/* 
+ * @Author: Abderrezzaq Sendjasni
+ * @Email: sendjasni.abderrezzaq@gmail.com
+ *
+ * Matrix multiplication with execution time recording  
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+
+const int MAT_SIZE = 500;
+
+void MatrixMultiplication()
```

Comment:
* Write this function only for matrix-multiplication. Not more!
* Write tests in the main-function

Reply:
Ok, thanks for the reply, I'll do that.